.github/workflows/ci.yaml

```diff
@@ -181,3 +181,52 @@ jobs:
                 env:
                     DATABASE_URL: "mysql://mysql:mysql@127.0.0.1:${{ job.services.mysql.ports[3306] }}/testdb"
                     DATABASE: "mysql://mysql:mysql@127.0.0.1:${{ job.services.mysql.ports[3306] }}/testdb"
+
+    solr-integration:
+        name: "Solr integration tests"
+        runs-on: "ubuntu-20.04"
+        timeout-minutes: 30
+        permissions:
+            packages: read
+            contents: read
+        strategy:
+            fail-fast: false
+            matrix:
+                php:
+                    - '7.4'
+                    - '8.0'
+                    - '8.1'
+        steps:
+            -   name: Start redis container
+                run: >- 
+                    docker run 
+                    --name redis 
+                    -p 6379:6379 
+                    -d 
+                    --entrypoint redis-server redis --appendfsync no --maxmemory 60mb --save ""
+    
+            - uses: actions/checkout@v2
+              with:
+                  fetch-depth: 0
+
+            - name: Setup PHP Action
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: 7.4
```

Comment:
Matrix defines this as tests against multiple PHP versions, yet here we are with a single one.

I guess it's an oversight?

Reply:
Yep, thank you! Will be fixed